pkgs/by-name/gn/gnucobol/package.nix

```diff
@@ -0,0 +1,135 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoconf,
+  automake,
+  libtool,
+  # libs
+  cjson,
+  db,
+  gmp,
+  libxml2,
+  ncurses,
+  # docs
+  help2man,
+  texinfo,
+  texliveBasic,
+  # test
+  perl,
+}:
+
+let
+  nistTestSuite = stdenv.mkDerivation {
+    pname = "gnucobol-nist-test-suite";
+    version = "foo";
```

Comment:
Maybe use the date? `unstable-xxxx-xx-xx`

Reply:
Good idea. Using the date of upload of that file. I'll research an authoritative date in the meantime.